docs/src/data/mockServices.tsx

```diff
@@ -32,33 +20,34 @@ export const mockServices = {
   }: {
     username: string;
     password: string;
-  }): Promise<any> {
-    if ((verifiedUsers.get(username).password === password) ||
-        (username === fixed_username && password === fixed_password)) {
+  }) {
+    if (
+      verifiedUsers.get(username).password === password ||
+      (username === FIXED_USERNAME && password === FIXED_PASSWORD)
+    ) {
       return Promise.resolve(verifiedUsers.get(username));
     } else {
-      return Promise.reject(new Error("Invalid username or password"));
+      return Promise.reject(new Error('Invalid username or password'));
```

Comment:
Do you plan to remove these errors too?

Reply:
I don't think so because that'll defeat the purpose of having fixed username and password combinations to preserve the authenticator flow.

Reply:
I'm good with leaving some basic error in place for the purpose of demo.